client/my-sites/plugins/plugin-details-header/test/index.jsx

```diff
@@ -51,11 +49,38 @@ describe( 'PluginDetailsHeader', () => {
 		plugin,
 	};
 
+	const initialState = {
```

Comment:
Non-blocking question:

I'm not sure I'm 100% up to date on recommended testing practices, but I'm curious about the switch here from mocking individual selectors to creating an entire state tree. Is one style more widely advised over the other, or is this more a preferential thing?

Reply:
I made this change to fix the failing test case.

But I think this can be done in a cleaner way: https://github.com/Automattic/wp-calypso/pull/81712/commits/cc6a3a7ec3d1a8a0ae67b8c5ff08634995b27da6

This method can be used if we need a specific set of values for the state values.

I have made the changes here: https://github.com/Automattic/wp-calypso/pull/81712/commits/cc6a3a7ec3d1a8a0ae67b8c5ff08634995b27da6